man/tpm2_policypcr.1.md

```diff
@@ -0,0 +1,93 @@
+% tpm2_policypcr(1) tpm2-tools | General Commands Manual
+%
+% JANUARY 2018
+
+# NAME
+
+**tpm2_policypcr**(1) - Perform a policyPCR event with the TPM.
+
+# SYNOPSIS
+
+**tpm2_policypcr** [*OPTIONS*]
+
+# DESCRIPTION
+
+**tpm2_policypcr**(1) generates a policy PCR event with the TPM. It is similair
+to **tpm2_createpolicy**(1), however, it expects a session to be already
+established via **tpm2_startauthsession**(1).
+
+# OPTIONS
+
+  * **-f**, **--policy-file**=_POLICY\_FILE_:
+    File to save the policy digest.
+
+  * **-F**, **--pcr-input-file**=_PCR\_FILE_:
+    Optional Path or Name of the file containing expected pcr values for the
+    specified index. Default is to read the current PCRs per the set list.
+
+  * **-L**, **--set-list**=_PCR\_LIST_:
+    The list of pcr banks and selected PCRs' ids (0~23) for each bank.
+
+  * **-S**, **--session**=_SESSION_FILE_:
+
+    The policy session file generated via the **-S** option to
+    **tpm2_startauthsession**(1).
+
+[common options](common/options.md)
+
+[common tcti options](common/tcti.md)
+
+[supported hash algorithms](common/hash.md)
+
+[algorithm specifiers](common/alg.md)
+
+# EXAMPLES
+
+Start a *trial* session and build a PCR policy and use that policy in an object.
```

Comment:
I think you can remove one "and", i.e: Start a *trial* session, build a PCR policy and use that policy in an object.